doc/api/crypto.md

```diff
@@ -937,8 +937,8 @@ The `private_key` argument can be an object or a string. If `private_key` is a
 string, it is treated as a raw key with no passphrase. If `private_key` is an
 object, it is interpreted as a hash containing two properties:
 
-* `key` : {String} - PEM encoded private key
-* `passphrase` : {String} - passphrase for the private key
+* `key` : {string} - PEM encoded private key
+* `passphrase` : {string} - passphrase for the private key
```

Comment:
Can you drop the spaces before the colons in this file? We don’t do that anywhere else in the docs